### ISSUE TITLE
sql: implement ALTER TABLE LOCALITY REGIONAL BY ROW -> !(REGIONAL BY ROW)

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#10077]
PR: https://github.com/cockroachdb/cockroach/pull/60311

From release notes:
> Implemented functionality to [`ALTER TABLE SET LOCALITY`](../v21.1/alter-table.html) from `REGIONAL BY ROW` to `REGIONAL BY TABLE` or `GLOBAL`. [#60311][#60311] {% comment %}doc{% endcomment %}

**Comment by ericharmeling** [comment]
@rmloveland
I don't think we need to document this any more than you already have done in https://www.cockroachlabs.com/docs/v21.1/set-locality.html ?

**Comment by rmloveland** [comment]
Eric: Yes!  I think all of the "implement going from A to B" issues can be closed.

Where A and B are one of the table localities we document on the linked page you provided
